### DISCUSSION TITLE
Taskbar on the left / right side of the screen

### DISCUSSION THREAD

**Discussion by neilroy1998** [lawand-dot-io/taskbar#9] [Feature requests]
This helps a lot in smaller screens like macbook airs, especially saving the vertical space!

**Comment by lawand-dot-io** [comment]
This is definitely an idea that I have in mind, but I have to find a way to clarify which window is which. I can't show the window title if the taskbar is on the right of left of the screen. I gotta think about this :)

**Comment by Ansssss** [comment]
+1 for this feature! Not sure what you were saying about clarifying which window is which though. Here's an example of a good vertical task bar on Windows 11 using an addon.

![Screenshot 2024-04-24 103327](https://github.com/lawand-dot-io/taskbar/assets/2245903/4860978f-3cf1-4cfc-949d-011decf62ccb)

**Reply by lawand-dot-io** [reply]
Interesting way to do it. I always assumed if I put Taskbar to the left or the right that I would have to keep the same dimensions, which means I can only show each window's icon and not title. Thanks, I will look into this at some point. Cheers

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/discussions/91

**Comment by rrisk93** [comment]
I really need this feature
Vertical windows taskbar is the best task manager IMO.
It saves vertical space in wide monitors like 16 : 9, and can view many tasks simultaneously at once. (if you set smaller icon in setting) 

I can tell which window is which if there is about 3 letters like below image.
![dcsd](https://github.com/user-attachments/assets/d65d1888-2322-495d-b115-426997d26b75)

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/discussions/176
